CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# CHANGELOG.md
+
+## v1.0.0
```

Comment:
If we are going to maintain app changelog as 1.0.0 then every new feature should be 1.x (Like navbar) and sub changes into it should be 1.x.x (like 11.1.aligning nav links 1.1.2. position of GitHub icon/rds logo). Just a suggestion.

Reply:
We'll be following [semantic versioning](https://semver.org/):
```
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```

For example, v1.2.3, here is 1 stands for MAJOR, 2 for MINOR and 3 for PATCH

Reply:
For more info: https://www.baeldung.com/cs/semantic-versioning

Reply:
With nice examples and easy to understand explanation: https://medium.com/fiverr-engineering/major-minor-patch-a5298e2e1798